doc/setup/quickstart_cmake/index.md

```diff
@@ -126,18 +126,19 @@ The output of the command `tree -L 2` should now look like this:
 .
 ├── build
 ├── sharg-parser
-│   ├── build_system
 │   ├── CHANGELOG.md
+│   ├── CMakeLists.txt
 │   ├── CODE_OF_CONDUCT.md
 │   ├── CONTRIBUTING.md
-│   ├── doc
-│   ├── include
 │   ├── LICENSE.md
 │   ├── README.md
+│   ├── build_system
+│   ├── doc
+│   ├── include
```

Comment:
fthe order depends on your system I guess. For me the order was correct because my OS was listing everything alphabetically without grouping directories.
Doesn't matter much I guess.

Reply:
Doesn't matter too much, I think. But there were a few files missing.